src/vm/corhost.cpp

```diff
@@ -108,7 +108,6 @@ CorHost2::CorHost2()
 {
     LIMITED_METHOD_CONTRACT;
 
-    m_fStarted = FALSE;
```

Comment:
Delete `m_fStarted` flag in the header as well?

Reply:
I am thinking the answer might be to make `Start()` and `Stop()` be NOPs.  Moving `InitializeEE()` into `CreateAppDomainWithManager`.  It would make startup essentially atomic.  It would remove a few state variables.  It would remove the reference counting which can prevent `UnloadAppDomain*` from calling `EEShutDown()`

Reply:
BTW, I deleted `m_fStarted` being initialized here because it is a member of the base class and is initialized in the base class constructor.

Reply:
> am thinking the answer might be to make Start() and Stop() be NOPs.

`Stop()` is pretty much no-op already. 

You can make Start() to just call `EnsureEEStarted` (that is called by `InitializeEE` today). `EnsureEEStarted` should be handle calls from multiple threads fine. Or if you would like to rather make it no-op - that would be fine with me too.

Reply:
I pushed a change to move the variable which belong in CorHost2 as Vitek suggested.

I looked at making Start() a nop.  Still felt messy.  I am leaving it for now.  There are more important things to work on...